supervisor/host/network.py

```diff
@@ -279,15 +284,15 @@ def from_dbus_interface(inet: NetworkInterface) -> Interface:
                 inet.connection.ipv4.nameservers,
             )
             if inet.connection and inet.connection.ipv4
-            else None,
```

Comment:
It feels to me nicer to return None, but if its a hassle for frontend maybe returning an empty is easier? @ludeeus

Reply:
Not a hassle, but this is better IMO.

Reply:
I agree, as we not represent NM on our abstracted interface, it's better to show what it currently is